.platform.app.yaml

```diff
@@ -33,6 +33,8 @@ hooks:
     curl -sS https://platform.sh/cli/installer | php
 
     pip install -r requirements/production.txt
+  deploy: |
+    PYTHONPATH=. alembic upgrade head
```

Comment:
I’ll try to double check this, but I think we can avoid setting the PYTHONPATH

Reply:
without the `PYTHONPATH=.` `import pyslackersweb` fails in `env.py`. I agree that there should be a better way but nothing I found so far.